Zpp/Common/ProviderDomain/Wrappers/ProductionOrderOperation.cs

```diff
@@ -175,6 +182,7 @@ public DueTime GetDueTime(IDbTransactionData dbTransactionData)
             DueTime dueTime;
             if (_productionOrderOperation.EndBackward != null)
             {
+                // TODO why end BackWard not just END ? 
```

Comment:
I changed this in 0.2. This is very important, since we must differentiate between the scheduled time (by backwards scheduling) and the job shop machine planning. Latter can produce times on operations that are not fitting to the times in demandToProviderGraph. E.g. an operation can start later and then it could be end later than the dueTime of its parent (e.g. stockExchangeProvider) --> which results in failing tests (time must decrease from top to bottom).

Reply:
Right, i changed it.